### PR TITLE
Header and footer style created

### PR DESCRIPTION
**Pull request by Monika39-maker**
Please check and suggest as needed

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -4,24 +4,22 @@ import About from "./pages/About";
 import Donations from "./Donations";
 import ThankYou from "./pages/ThankYou";
 import Home from "./pages/Home";
-import About from "./pages/About";
```
Comment (HThanki) [inline client/src/App.js:7]: Checking should this be removed if we are using about component in our react routes?

**Review by HThanki — COMMENTED** [review]
PR reviewed but not merged. Looks good but had one query. Added one comment around checking if import about line in App.js should or should not be remove if being used in our react routes

File: client/src/App.js (hunk 1 above)
Comment (HThanki) [reply client/src/App.js:7]: Hi @Monika39-maker, I don't think this is resolved. Please could you take a look

**Review by DeagoBrowne — APPROVED** [review]
I'm going to merge everything, then go through it all for clean up, and make a new PR

**Pull request merged**